v4/context/context.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2021 PingCAP, Inc. Licensed under Apache-2.0.
+// This code is copied from https://github.com/pingcap/dm/blob/master/pkg/context/context.go
+
+package context
+
+import (
+	"context"
+	"time"
+
+	"github.com/pingcap/dumpling/v4/log"
+)
+
+// Context is used to in dm to record some context field like
+// * go context
+// * logger
+type Context struct {
```

Comment:
how about

```
type Context struct {
	context.Context
	logger log.Logger
}
```

so we don't need `(*Context).Context()`. and logger could only be accessed by `L()`

Reply:
addressed in https://github.com/pingcap/dumpling/pull/242/commits/fcc2ab6e9f673aa3a09baea0d4cae588cd9fbaaa